test/integration/model/merge_include_with_interface_api_1.sdf

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<sdf version="1.10">
+  <model name="intermediate_model">
+    <include>
```

Comment:
should this file name really start with `merge_include_` when its embedded `include` tag doesn't have `@merge` set to true?

Reply:
Renamed to `intermediate_model_with_interface_api_1.sdf` in https://github.com/gazebosim/sdformat/pull/1293/commits/e852813155496fa72b3d4d9b530f1200a5195142